capstone/src/main/java/com/google/univiz/collegescorecarddata/CollegeScoreCard.java

```diff
@@ -0,0 +1,19 @@
+package com.google.univiz.collegescorecarddata;
+
+import java.util.List;
+
+public interface CollegeScoreCard {
+  /** General Use Functions */
```

Comment:
what does this comment mean? what would non-general user function be?

Reply:
update comments!